testnet_bitcoin_wallet/stx.py

```diff
@@ -65,12 +66,17 @@ def multi_send(username):
     try:
         for i in range(num_r):
             target_address = input(f"Recipient {i+1}: ")
+            prefix = target_address[0]
             try:
-                target_h160 = decode_base58(target_address)
+                if prefix == "m" or prefix == "n":
+                    target_h160 = decode_base58(target_address)
+                    target_script = p2pkh_script(target_h160)
+                else:
+                    target_h160 = decode_bech32(target_address, testnet=True)
+                    target_script = make_p2wpkh_script(target_h160)
```

Comment:
I don't think you were aiming to resolve that already, but think about what would happen here if `target_address` were a P2WSH or a P2TR address in `decode_bech32(target_address, …)`

Reply:
Yeah, this would actually work with a P2WSH address but with P2TR I think it will just make a P2WPKH or P2WSH script using the information encoded in the address, which is dangerous.